commands/Music/clearqueue.js

```diff
@@ -0,0 +1,36 @@
+const Command = require("../../base/Command");
```

Comment:
These commands aren't needed like I said. They won't work unless queues exists, so if you delete queues from Comfy.js, then these need to go too